.github/workflows/ci.yaml

```diff
@@ -1,73 +1,54 @@
 name: ci
 on:
   push:
-    branches: [main]
+    branches:
+      - main
   pull_request:
-    branches: [main]
-  schedule:
-    - cron: '15 22 * * *'
-  workflow_dispatch: {} # support manual runs
-# Prevent writing to the repository using the CI token.
-# Ref: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions
-permissions: read-all
+    types:
+      - opened
+      - synchronize
+      - reopened
 env:
-  SERVICE: connect-demo
-  REGISTRY: connect-demo
-  REGION: us-central1
+  MAKEFLAGS: "-j 8"
+  PRIVATE_REGISTRY_HOST: us.gcr.io
+  PRIVATE_REGISTRY_URL: us.gcr.io/buf-private-gcr-1
+permissions:
+  contents: read
 jobs:
-  ci:
+  testlint:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout Code
+      - name: checkout
         uses: actions/checkout@v3
-        with:
-          fetch-depth: 1
-      - name: Install Go
+      - name: setup-go
+        if: success()
```

Comment:
I'm fairly sure that this is the default, and that we should remove it from any workflows that include it.

Reply:
cleared